EN.yaml

```
@@ -511,7 +511,7 @@ PORTAL_CHATMESSAGE_TOO_EARLY: You will be able to move shortly.
 PSP_AEGIR: Aegir the Angry
 PSP_AMON: Overseer Amon
 PSP_AMORA: Amora
-PSP_ARCHDAEMON_AMON: Archdaemon Amon
+PSP_ARCHDAEMON_AMON: Archdemon Amon

```

it's Archdaemon on official for some reason, see: daemon (classical mythology)
```suggestion
PSP_ARCHDAEMON_AMON: Archdaemon Amon

```